### PR TITLE
Create footer

### PR DESCRIPTION
**Pull request by GabrielBFurlani**
footer-preview

Closes #19

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -333,15 +337,129 @@ <h2 class="featurette-heading-social-media">
...
+                alt="Github"
+            /></a>
+          </p>
+        </div>
+      </div>
+    </div>
+
+    <!-- FOOTER -->
+    <footer class="low-footer">
+      <div>&copy; 2016 - 2018 | ProFUSION Sistemas e Soluções LTDA</div>
```
Comment (victor-accarini) [inline index.html:462]: 2016-2021

File: index.html [hunk 2]
```diff
@@ -333,15 +337,129 @@ <h2 class="featurette-heading-social-media">
...
+            <p><a class="footer-link" href="#contact-us">Contact</a></p>
+          </div>
+        </div>
+        <div class="grid-footer-element-4-second bottom-space-5">
+          <div class="top-space-5">
+            <p>
+              <span style="color: #63c3d1"> ADDRESS </span>
+              <br />
+              Av. Engenheiro Carlos Stevenson, nº 80 <br />
+              Sala 51 - 13092-132 - Campinas <br />
```
Comment (victor-accarini) [inline index.html:372]: Sala 11

File: index.html [hunk 3]
```diff
@@ -333,15 +337,129 @@ <h2 class="featurette-heading-social-media">
...
+            </p>
+            <br />
+            <br />
+            <p>
+              <span style="color: #63c3d1"> CONTACT </span>
+              <br />
+              +55 19 33221144 / +55 19 33445566 <br />
+              sales@profusion.com.br / jobs@profusion.com
```
Comment (victor-accarini) [inline index.html:380]: Here I would put `profusion.mobi` in both of the emails...

@barbieri is that correct?

**Review by victor-accarini — COMMENTED** [review]
We have two commits in this PR, one saying "incomplete footer" and another "create footer", each commit should ideally be a working piece of code, so the first commit "incomplete footer" doesn't make much sense since it's not a "finished" work, it would be better to have only one commit with everything.

If you wanted to split the work you could have added the assets in the first commit and the code in the second, that would be better.

**Comment by victor-accarini** [comment]
Could you also fix the `social-media-medium.PNG` file to have a lowercase extension? So it doesn't break when looking in case sensitive operational systems...

File: index.html (hunk 3 above)
Comment (GabrielBFurlani) [reply index.html:380]: Dilly sent it to us in shared-dev, thanks

**Review by victor-accarini — APPROVED** [review]

**Comment by GabrielBFurlani** [comment]
Closes #19

**Pull request merged**